tests/unittests/test_utilities.py

```diff
@@ -342,9 +343,9 @@ def test_get_sdk_from_sys_path_after_updating_sys_path(self):
         """
         sys.path.insert(0, self._dummy_sdk_sys_path)
         module = common.get_sdk_from_sys_path()
-        self.assertEqual(
+        self.assertNotEqual(
```

Comment:
Customer not bringing azure-functions needs to be verified

Reply:
This is not unit testable. Will try to add an e2e test later